train-A-app/src/app/admin-overview/components/route-item/route-item.component.html

```diff
@@ -2,7 +2,7 @@
   <div class="item__header">
     <h2 class="item__title">Route {{ item.id }}</h2>
     <div class="item__buttons">
-      <div class="item__button" (click)="onUpdate()">
+      <div class="item__button" (click)="onUpdate()" (keydown.enter)="onUpdate()" tabindex="0">
```

Comment:
I've changed this component. During merge, please, apply my changes. The same with all buttons